File: Alarm_Clock/Alarm.cpp
#include <stdio.h>
#include <unistd.h>  // for sleep()

int main() {
    int seconds;
    printf("Enter time in seconds: ");
    scanf("%d", &seconds);

    printf("Timer set for %d seconds. Starting now...\n", seconds);

    while (seconds > 0) {
        int minutes = seconds / 60;
        int secs = seconds % 60;
        printf("\r%02d:%02d", minutes, secs);
        fflush(stdout);  // force print immediately
        sleep(1);
        seconds--;
    }

    printf("\n⏰ Time's up!\n");

    return 0;
}

File: Alarm_Clock/Alarm.py
import time

def simple_timer():
    seconds = int(input("Enter time in seconds: "))
    print(f"Timer set for {seconds} seconds. Starting now...")

    while seconds:
        mins, secs = divmod(seconds, 60)
        timer_display = f"{mins:02d}:{secs:02d}"
        print(timer_display, end="\r")
        time.sleep(1)
        seconds -= 1

    print("\n⏰ Time's up!")

if __name__ == "__main__":
    simple_timer()
